tests/integration/test_identity.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+# cunumeric.identity(n, dtype=None, *, like=None)
+
+DTYPE_ALL = [
+    "l",
+    "L",
+    "f",
+    "e",
+    "d",
+    "h",
+    "i",
+    "H",
+    "I",
+    "?",
+    "b",
+    "B",
+    "F",
+    "D",
+]
```

Comment:
Do we not define a similar list in a central place somewhere that could be re-used?